chains/solana/contracts/programs/ccip-offramp/src/lib.rs

```diff
@@ -171,6 +172,7 @@ pub mod ccip_offramp {
         router: Pubkey,
         fee_quoter: Pubkey,
         offramp_lookup_table: Pubkey,
+        rmn_remote: Pubkey,
```

Comment:
Whoops, I just saw that the docstring for this method was outdated (even before this PR). Could you please update it?